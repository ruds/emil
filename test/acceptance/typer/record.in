{foo = 2, bar = "hi there", baz = 2.3};
{a = it, b = "that's a biggie"};
{foo = 23i : int};
{foo = 23i : string};
