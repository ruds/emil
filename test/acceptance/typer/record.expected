@0001:
val it : {bar: string, baz: float, foo: bigint}
@0002:
val it : {a: {bar: string, baz: float, foo: bigint}, b: string}
@0003:
val it : {foo: int}
@0004: ERROR
@0005:
